Dockerfile

```diff
@@ -1,4 +1,23 @@
+FROM node:6.9.5 as app
+
+WORKDIR /app
+
+#wildcard as some files may not be in all repos
+COPY package*.json npm-shrink*.json /app/
+
+RUN npm install --quiet
+
+COPY . /app
+
+
+RUN npm run compile:all
+
 FROM node:6.9.5
 
-# we also need imagemagick but it is already in the node docker image
-RUN apt-get update && apt-get install -y --no-install-recommends ghostscript optipng
```

Comment:
Any reason we are switching from installing `ghostscript` from a package to compiling from source?  If we used an appropriate base image we could get security updates for ghostscript automatically.